apps/wing/src/commands/test.ts

```diff
@@ -153,6 +158,65 @@ async function testSimulator(synthDir: string) {
   }
 }
 
+async function testAwsCdk(synthDir: string): Promise<sdk.cloud.TestResult[]> {
+
+  if (!isAwsCdkInstalled(synthDir)) {
+    throw new Error(
+      "AWS-CDK is not installed. Please install AWS-CDK to run tests in the cloud (npm i aws-cdk)."
+    );
+  }
```

Comment:
When I tried this out (without CDK installed), I didn't see the error message and instead received:

```
✔ Compiling to awscdk...
⠋ cdk deploynode:internal/errors:867
  const err = new Error(message);
              ^

Error: Command failed: cdk version
/bin/sh: cdk: command not found

    at ChildProcess.exithandler (node:child_process:419:12)
    at ChildProcess.emit (node:events:513:28)
    at maybeClose (node:internal/child_process:1091:16)
    at Socket.<anonymous> (node:internal/child_process:449:11)
    at Socket.emit (node:events:513:28)
    at Pipe.<anonymous> (node:net:322:12) {
  code: 127,
  killed: false,
  signal: null,
  cmd: 'cdk version',
  stdout: '',
  stderr: '/bin/sh: cdk: command not found\n'
}

Node.js v18.14.1
```

You might need to try/catch inside of `isAwsCdkInstalled` in order to show the error message correctly.

Reply:
Done, thanks @Chriscbr